Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/services/GameFileCacheService.java

```diff
@@ -86,6 +86,11 @@ public static GameFile findSecondDisc(GameFile game)
     return matchWithoutRevision;
   }
 
+  public static int GetDiscNumber(GameFile game)
```

Comment:
There is no reason for this method to exist. Calls to it can be replaced with `game.getDiscNumber()`.